src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix that filters flags
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
+    {
+        $constants = get_defined_constants();
+        foreach ($constants as $c => $v) {
+            if ('' !== $prefix && 0 !== strpos($c, $prefix) || ($value & $v) !== $v) {
+                unset($constants[$c]);
+            }
+        }
+
+        asort($constants);
+        $name = implode(' | ', array_flip($constants)) ?: 0;
```

Comment:
well, compute the value of the flag based on selected constants, and ensure it is equal to the dumped flag. If it is not equal, it means that the dumped flag contains more bits)